src/Symfony/Component/ExpressionLanguage/Tests/ParserTest.php

```diff
@@ -161,4 +161,44 @@ private function createGetAttrNode($node, $item, $type)
     {
         return new Node\GetAttrNode($node, new Node\ConstantNode($item), new Node\ArgumentsNode(), $type);
     }
+
+    /**
+     * @expectedException \Symfony\Component\ExpressionLanguage\SyntaxError
+     */
+    public function testParseWithStringAsInvalidName()
+    {
+        $lexer = new Lexer();
+        $parser = new Parser(array());
+        $parser->parse($lexer->tokenize('foo."#"'), array('foo'));
+    }
```

Comment:
I guess the travis issue can be ignored because the ExpressionLanguage does not have any travis configuration file, but still don't know how to fix the "fabbot.io" issue. 
I believe that's because I squashed the commits and used "git push --force ..." to replace the remote branch. Maybe I can close this one and raise a new pull request.